lovesay/cli.py

```diff
@@ -1,18 +1,20 @@
 from typing import Optional
 import typer
 import lovesay.love as lovesay
+import shutil
```

Comment:
Since `shutil` is now being imported in this script, it can be safely removed from the `love.py`